operator/v1/types_etcd.go

```diff
@@ -28,12 +28,33 @@ type Etcd struct {
 
 type EtcdSpec struct {
 	StaticPodOperatorSpec `json:",inline"`
+	// profile allows user to change the etcd tuning profile which configures
+	// the latency parameters for heartbeat interval and leader election timeouts
+	// allowing the cluster to tolerate longer round-trip-times between etcd members.
+	// Valid values are "Default" and "Slower".
+	// The current default is Default.
+	// +kubebuilder:validation:Optional
+	// +openshift:enable:FeatureSets=CustomNoUpgrade;TechPreviewNoUpgrade
+	// +optional
+	Profile ProfileValue `json:"profile,omitempty"`
```

Comment:
There's no `// +kubebuilder:default:=Default`. Is the default value managed by the etcd operator? Is the default value a subject to change?

Reply:
How will the switching between Default and Slower will be handled? Any danger to the etcd instances when the heartbeat interval and leader election timeouts are changed? Any danger when heartbeat interval and leader election timeouts are changed to quickly (i.e. malicious user/admin alternating the values)?

Reply:
I'll add the default with the kubebuilder.

The current plan is to require an etcd rollout to consume the changed value. A rollout will restart each etcd instance one-by-one so there's no risk hot-changing the value.

We're still considering not requiring the rollout, but that will be for a future change.